local-volume/provisioner/pkg/deleter/jobcontroller.go

```diff
@@ -289,7 +305,8 @@ func NewCleanupJob(pv *apiv1.PersistentVolume, imageName string, nodeName string
 
 	// Annotate job with useful information that cannot be set as labels due to label name restrictions.
 	annotations := map[string]string{
-		DeviceAnnotation: blkdevPath,
+		DeviceAnnotation:    blkdevPath,
+		StartTimeAnnotation: time.Now().Format(time.RFC3339Nano),
```

Comment:
I think the Job object already has its own start time. Can that be used?

Reply:
@msau42 job `status.startTime` is the time when job was acknowledged by the job controller. It does not include network latency, apiserver/controller-manager process latency. If apiserver/controller-manager failed to process the job in time, we couldn't know the latency through these metrics.